Description: set qmldir.path correctly
Author: Dmitry Shachnev <mitya57@gmail.com>
Forwarded: yes, https://codereview.qt-project.org/69586
Last-Update: 2013-10-29

--- a/examples/sensors/grue/import/import.pro
+++ b/examples/sensors/grue/import/import.pro
@@ -21,7 +21,7 @@
 INSTALLS += target
 
 qmldir.files=$$PWD/qmldir
-qmldir.path=$$DESTDIR
+qmldir.path=$$DESTPATH
 INSTALLS += qmldir
 
 OTHER_FILES += \
